themes/default/asset-master-pack.json

```diff
@@ -18,85 +18,5 @@
             "urls": ["shared/button-click.mp3", "shared/button-click.ogg"],
             "autoDecode": true
         }
```

Comment:
Maybe I am overthinking it but does it seem a bit strange to have this as the only thing left in asset-master-pack.json?

Reply:
Would everything implode if it wasn't there? If it isn't required. could probably be renamed to `loadscreen.json`. If it is required then, yeah...  ¯\_(ツ)_/¯

Now that I think about it though that whole block is weird anyway because none of those audio files are really "loadscreen" assets. I think me and Sean basically had to just shoehorn them in somewhere and I am not sure they load if they are listed under some screen or other.